.travis.yml

```diff
@@ -34,26 +38,29 @@ before_install:
     export MYSQL_URL="mysql+mysqlconnector://root@localhost/utc_test";
   fi
 - export TEST_DATABASE_URLS="$PG_URL $MYSQL_URL"
+
 install:
 - if [[ "$PIP_OPTS" != "" ]]; then
     pip install $PIP_OPTS "SQLAlchemy $SQLALCHEMY_VER";
   else
     pip install "SQLAlchemy $SQLALCHEMY_VER";
   fi
 - pip install -e .
-- pip install pytest flake8 "import-order >= 0.0.10"
+- pip install pytest flake8 flake8-import-order
```

Comment:
This change should be reflected to tox.ini as well (the duplication per se is problematic though).

Reply:
I'll see what I can do.

Reply:
`tox.ini` is up to date now.

I will come back to unifying tox and travis in a later PR.